powerline/segments/vim/plugin/ale.py

```diff
@@ -0,0 +1,97 @@
+# vim:fileencoding=utf-8:noet
+"""
+Asynchronous Lint Engine (ALE) plugin
+
+Largely adapted from Sytastic plugin
+"""
+
+from __future__ import (
+	unicode_literals, division, absolute_import, print_function
+)
```

Comment:
Just copy the header from syntastic: no wrapping, more empty lines and no docstring.

Reply:
Fixed in new commit